lib/screens/collective/perspectives/expression_feed.dart

```diff
@@ -55,10 +56,11 @@ class _ExpressionFeedState extends State<ExpressionFeed> {
               ),
               pinned: false,
               floating: true,
-            ), 
+            ),
           ],
           body: CollectiveFeedRefresh(
             child: CustomScrollView(
+              shrinkWrap: true,
```

Comment:
Are you sure this is the only way?

> Shrink wrapping the content of the scroll view is significantly more expensive than expanding to the maximum allowed size because the content can expand and contract during scrolling, which means the size of the scroll view needs to be recomputed whenever the scroll position changes.

Reply:
The scroll view needs to have a size in order to achieve the scrolling header effect. The alternative would be to use an Intrinsic widget however this does something similar under the hood.